### PR TITLE
Add testcase for https://github.com/wso2/product-ei/issues/1180.

### PR DESCRIPTION
**Pull request by vinok88**

### CODE REVIEW

File: integration/mediation-tests/tests-transport/src/test/java/org/wso2/carbon/esb/jms/inbound/transport/test/JMSInboundMessagePollingTestCase.java [hunk 1]
```diff
@@ -111,6 +133,45 @@ public void destroy() throws Exception {
...
+			for (int i = 0; i < 5; i++) {
+				sender.pushMessage(message);
+			}
+		} finally {
+			sender.disconnect();
+		}
+
+		Thread.sleep(5000);
+
+		LogViewerClient logViewerClient = new LogViewerClient(contextUrls.getBackEndUrl(), getSessionCookie());
```
Comment (milindaperera) [inline integration/mediation-tests/tests-transport/src/test/java/org/wso2/carbon/esb/jms/inbound/transport/test/JMSInboundMessagePollingTestCase.java]: Let's retrieve log viewer client  and clear logs before sending the request

File: integration/mediation-tests/tests-transport/src/test/java/org/wso2/carbon/esb/jms/inbound/transport/test/JMSInboundMessagePollingTestCase.java [hunk 2]
```diff
@@ -45,6 +51,22 @@
 	private InboundAdminClient inboundAdminClient;
 	private ActiveMQServer activeMQServer = new ActiveMQServer();
 
+	String message = "<?xml version='1.0' encoding='UTF-8'?>" +
```
Comment (milindaperera) [inline integration/mediation-tests/tests-transport/src/test/java/org/wso2/carbon/esb/jms/inbound/transport/test/JMSInboundMessagePollingTestCase.java]: Let's add access modifiers
Do we really need to make this class variable? I see only used in one function? or you planned for ahead :)

Comment (vinok88) [reply integration/mediation-tests/tests-transport/src/test/java/org/wso2/carbon/esb/jms/inbound/transport/test/JMSInboundMessagePollingTestCase.java]: Just added for the clarity. Di we need to care of access modifiers? This is a test class.

File: integration/mediation-tests/tests-transport/src/test/java/org/wso2/carbon/esb/jms/inbound/transport/test/JMSInboundMessagePollingTestCase.java (hunk 1 above)
Comment (vinok88) [reply integration/mediation-tests/tests-transport/src/test/java/org/wso2/carbon/esb/jms/inbound/transport/test/JMSInboundMessagePollingTestCase.java]: It is done, check the line 159.

File: integration/mediation-tests/tests-transport/src/test/java/org/wso2/carbon/esb/jms/inbound/transport/test/JMSInboundMessagePollingTestCase.java [hunk 3]
```diff
@@ -111,6 +133,45 @@ public void destroy() throws Exception {
...
+		try {
+			sender.connect(queueName);
+			for (int i = 0; i < 5; i++) {
+				sender.pushMessage(message);
+			}
+		} finally {
+			sender.disconnect();
+		}
+
+		Thread.sleep(5000);
```
Comment (himasha) [inline integration/mediation-tests/tests-transport/src/test/java/org/wso2/carbon/esb/jms/inbound/transport/test/JMSInboundMessagePollingTestCase.java]: Can we add the sleep in a loop?

Comment (vinok88) [reply integration/mediation-tests/tests-transport/src/test/java/org/wso2/carbon/esb/jms/inbound/transport/test/JMSInboundMessagePollingTestCase.java]: It is not possible. We are waiting for messages to get pushed to ActiveMQ server. There is no way we can check whether is happened.

File: integration/mediation-tests/tests-transport/src/test/java/org/wso2/carbon/esb/jms/inbound/transport/test/JMSInboundMessagePollingTestCase.java (hunk 2 above)
Comment (milindaperera) [reply integration/mediation-tests/tests-transport/src/test/java/org/wso2/carbon/esb/jms/inbound/transport/test/JMSInboundMessagePollingTestCase.java]: Better adding for perfection, It doesn't matter test, dummy, sample, product, code :D

File: integration/mediation-tests/tests-transport/src/test/java/org/wso2/carbon/esb/jms/inbound/transport/test/JMSInboundMessagePollingTestCase.java (hunk 1 above)
Comment (milindaperera) [reply integration/mediation-tests/tests-transport/src/test/java/org/wso2/carbon/esb/jms/inbound/transport/test/JMSInboundMessagePollingTestCase.java]: Just understood what you trying to test in this test case.

File: integration/mediation-tests/tests-transport/src/test/java/org/wso2/carbon/esb/jms/inbound/transport/test/JMSInboundMessagePollingTestCase.java (hunk 3 above)
Comment (milindaperera) [reply integration/mediation-tests/tests-transport/src/test/java/org/wso2/carbon/esb/jms/inbound/transport/test/JMSInboundMessagePollingTestCase.java]: let's add comment mentioning for what we are waiting for

**Pull request merged**